packages/manager/modules/core/src/index.js

```diff
@@ -28,6 +28,7 @@ import redirectionService from './redirection/redirection.service';
 import managerCoreUrlBuilder from './url-builder';
 
 import { registerConfigModule } from './config';
+import secureUrl from './secure-url/secure-url';
```

Comment:
suggestion: 
```suggestion
import sanitizeUrl from './secure-url/secure-url';
```
is more explicit about what the module does

Reply:
changed